### ISSUE TITLE
Many stations do not have departure times

### ISSUE THREAD

**Issue by Alisamix** [patrickbr/simpleefa#2]
Many stations (but not all) dont have any departure times, while at the same time the older version of simpleefa shows the correct results.

``` json
{
 "for": {
  "station": {
   "station_name": "Rotkreuzplatz",
   "location_name": "München",
   "name": "München, Rotkreuzplatz",
   "id": "1000190",
   "position": {
    "lng": "11.53350943091",
    "lat": "48.152984108039995"
   }
  },
  "dateandtime": {
   "date": "2015-7-15",
   "time": "20:23:00"
  }
 },
 "id": "1"
}
```

I found out that the onlyTypes is set the 1 per default, so it seems. Adding the parameter onlyTypes=2 shows the U-Bahn connections. 1!2 still only shows S-Bahn connections.

**Comment by patrickbr** [comment]
Thanks for reporting this bug, I could successfully reproduce it and will post a solution for it during the evening.

**Comment by Alisamix** [comment]
Thank you for creating simpleefa :+1:

**Comment by patrickbr** [comment]
Should be fixed with the latest release (v0.3.6)
